scripts/install-open5gs.sh

```diff
@@ -8,14 +10,20 @@ fi
 
 PS4='[DEBUG] '
 set -x
-
+# \
+# [Docker](https://docs.docker.com/engine/install/ubuntu/)
```

Comment:
Add a newline after Docker?

Reply:
For some reason in CodeChat the linked references do not seem work. I would like to discuss them with you in our next meeting.